### PR TITLE
Add how to use inspector with node avo functions

### PR DESCRIPTION
**Pull request by tpom6oh**

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝Changed routes:


| [**…tor-with-avo-functions**](https://docs-git-node-avo-functions-and-inspector.teamavo.now.sh/implementation/start-using-inspector-with-avo-functions) |
|:-:|
|<a href="https://docs-git-node-avo-functions-and-inspector.teamavo.now.sh/implementation/start-using-inspector-with-avo-functions">Screenshot of /implementation/start-using-inspector-with-avo-functions</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/WTz9m5rx48C6fB_115DRg5jCiw4_.png">(view full size)</a> |







Commit 72d09aeb075a0d2dc6a86834c9674c3e40f2b882 (https://docs-git-node-avo-functions-and-inspector.teamavo.now.sh).

File: pages/implementation/start-using-inspector-with-avo-functions.mdx [hunk 1]
```diff
@@ -105,6 +115,47 @@ Avo.initAvo({ env: Avo.AvoEnv.Dev, inspector: inspector }, ...);
...
+```javascript
+const Inspector = require("node-avo-inspector");
+
+const inspector = new Inspector.AvoInspector({
+  apiKey: Avo.avoInspectorApiKey,
+  env: env.DEV,
+  version: "1.0.0",
+  appName: "My App"
+});
+Avo.initAvo( { env: env.DEV, inspector: inspector}, ...);
```
Comment (hugihlynsson) [inline pages/implementation/start-using-inspector-with-avo-functions.mdx:141]: Should this say Avo.AvoEnv.Dev like the typescript code? Should we add the Avo import to be explicit?

Are the Typescript and Javascript examples the same? Would it be enough to have just one of them?

File: pages/implementation/start-using-inspector-with-avo-functions.mdx [hunk 2]
```diff
@@ -105,6 +115,47 @@ Avo.initAvo({ env: Avo.AvoEnv.Dev, inspector: inspector }, ...);
 
 </details>
 
+---
+
+<details>
+<summary><strong>Node</strong></summary>
+<p/>
+To enable Avo Functions to report to Avo Inspector automatically you would need to provide an instance of Avo Inspector, <a target="\_blank" rel="noopener noreferrer" href="https://github.com/avohq/node-avo-inspector">check the quick start guide for how to create one</a>, when initializing Avo and use Avo version >= `71.34.0` and Avo Inspector SDK version >= `0.1.0`.
```
Comment (hugihlynsson) [inline pages/implementation/start-using-inspector-with-avo-functions.mdx:123]:
```suggestion
To enable Avo Functions to report to Avo Inspector automatically you provide an instance of the Inspector when initializing Avo. This requires Avo version >= `71.34.0` and Avo Inspector SDK version >= `0.1.0`. <a target="\_blank" rel="noopener noreferrer" href="https://github.com/avohq/node-avo-inspector">Check the quick start guide for how to create an Avo Inspector instance</a>.
```

**Review by hugihlynsson — APPROVED** [review]
Nice!

File: pages/implementation/start-using-inspector-with-avo-functions.mdx (hunk 1 above)
Comment (tpom6oh) [reply pages/implementation/start-using-inspector-with-avo-functions.mdx:141]: Good catch, this actually should be a string in JS

**Pull request merged**